clap_generate/src/generators/shells/fish.rs

```diff
@@ -71,6 +71,12 @@ fn gen_fish_inner(root_command: &str, app: &App, buffer: &mut String) {
 
         if let Some(data) = option.get_long() {
             template.push_str(format!(" -l {}", data).as_str());
+
+            if let Some(aliases) = option.get_visible_aliases() {
```

Comment:
It looks like the original `get_long` and `get_short` in all these files can be converted to use `get_longs_and_visible_aliases` and `get_shorts_and_visible_aliases`. That way, we can reduce the code duplication.